src/image_builder.py

```diff
@@ -139,6 +139,15 @@ def image_builder(buildspec):
         if "labels" in image_config:
             labels.update(image_config.get("labels"))
 
+        # Adding standard labels to all images
+        labels["com.amazonaws.sagemaker.dlc.framework"] = str(BUILDSPEC["framework"])
```

Comment:
Please add validation tests to ensure that the labels are applied correctly on all images that are built.